src/Symfony/Component/Validator/Resources/translations/validators.af.xlf

```diff
@@ -442,6 +442,14 @@
                 <source>This URL is missing a top-level domain.</source>
                 <target state="needs-review-translation">Die URL mis 'n topvlakdomein.</target>
             </trans-unit>
+            <trans-unit id="114">
+                <source>This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</source>
```

Comment:
In English, we should have a specific case for when min == 1, right?

Reply:
Isn’t this the part on the left side of the `|`?

Reply:
I missed the |, sorry